installation/local/rotate_neo4j_password.yml

```diff
@@ -0,0 +1,34 @@
+- hosts: neo4j[0]
+  tasks:
+    - name: Neo4j password test
```

Comment:
```suggestion
    - name: Generate new neo4j password and store it in pwsafe
```

Reply:
Amended the commit with an updated task name.